understanding/21/target-size-enhanced.html

```diff
@@ -19,6 +19,7 @@ <h2>Intent</h2>
 			  <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
 			  <li>the control is part of a sequential task.</li>
 	  </ul>
+	  <p>This SC does not require that the visible indicator of a target, such as a button or icon, be the same size as the target. It is recommended that when the visible indicator of the target is notably smaller than the target, that the visible indicator be placed at the center of the target.</p>
```

Comment:
I believe we always expand "SC" to "Success Criterion" (here and in the other file change)